config/routes.rb

```diff
@@ -6,5 +6,9 @@
   resources :people
   resources :home, only: [:index]
 
-  root to: "home#index"
```

Comment:
Why did we change the root to the dashboard? Shouldn't it be on the home? We want them to come to a landing page ie the home page and then log in to access the dashboard